### ISSUE TITLE
share executive powers between President and Executive Committee

### ISSUE THREAD

**Issue by nmeyne** [rchain-community/sm19a#2]
proposed during the May 4 session : to revise the RChain Executive committee to share executive powers with the President, noting interest from John B, mutineering (@nmeyne) in serving

a poll for members to express support is in [#governance 9:46am KC time](https://discordapp.com/channels/375365542359465989/393462637100400650/574245496646926350)

context: [discussion in 4 May session](https://docs.google.com/document/d/13qSmFAnYNhjnJY9lt-YBX01CBn8WhmqCnagoDIq6jM0/edit#heading=h.370e1m3wzcm9)

----
_The scope of this issue has been refined; originally..._

The Coop needs as much transparency as possible on the financing deal options on the table or in the pipeline, but must respect the confidentiality / privacy of potential investors.  A working group of trusted members (from candidate nominations and community voting) should be appointed. The committee members would operate under a limited non-disclosure agreement:  The committee should provide the membership with maximum possible transparency on the deals / potential deals available and should seek membership guidance on the deal assessment criteria.  Members of the committee will keep investor confidentiality.  The committee may seek expert advice if necessary.   Board members will participate in the committee process and are expected to endorse and enact the decisions of the committee.

**Comment by dckc** [comment]
This sounds a lot like the audit and finance committee, to me. I can see some distinctions, but does it seem cost-effective to have two separate groups? Perhaps we could revise the charter of the AFC to cover this?

Regardless, I do hope to see offers to participate in this, whatever form it takes.

I don't think it's a good match for my talents.

**Comment by nmeyne** [comment]
Yes, good point....  (and I think you're being modest about your talents).   I think it's maybe a temporary extension of the AFC remit...    more active decision making than post-hoc **Audit** of some board decision and covering more than just **Financial** criteria.  Ethics / relationships anyone?

**Comment by dckc** [comment]
Another possibility:

> The role of the Executive Committee is to implement the Board's fiduciary, strategic, and generative plans, policies, and decisions consistent with the Cooperative's vision, mission, and guiding principles.

**Comment by dckc** [comment]
I'm re-scoping this to align with the proposal generated in this morning's session:

proposed (1) : to revise the RChain Executive committee to share executive powers with the President, noting interest from John B, mutineering (@nmeyne) in serving

It currently has support from 10 memebers:
[#governance 9:46am KC time](https://discordapp.com/channels/375365542359465989/393462637100400650/574245496646926350)

**Comment by justyhoo** [comment]
Hi Nick and Dan, as mentioned to Rich, I’m happy to assist with your proposed temporary extension of the remit of the AFC addressing ethics/relationships of potential deals. Would be keen to get some further guidance from Nick as to what he envisions for this task. Thanks, Justin

**Comment by richjensen3000** [comment]
@justyhoo I would like to collaborate with you and any other qualified Members to re-activate the AFC in lieu of Mark's departure. Would you be free to discuss potential agenda items toward an AFC meeting this week?\

**Comment by justyhoo** [comment]
@richjensen3000 yes, probably best placed to meet around 2pm your time if that suits you in the next couple of days.

**Comment by dckc** [comment]
More help is of course welcome, but we do have multiple offers to participate here.

**Comment by richjensen3000** [comment]
Update: @justyhoo  and I had an informal conversation yesterday afternoon, also attended by @ddayan and John Bodin, identifying tasks toward reactivation of AFC.  Mark Pui has offered advice and assistance to this effort, suggesting a first step of recruiting a Director from the Coop's Board to sit on the AFC.  Pursuing that now. Speak up, especially if you have legal or accounting experience and would like to be considered for a Committee seat or observer role.

**Comment by richjensen3000** [comment]
> This sounds a lot like the audit and finance committee, to me. I can see some distinctions, but does it seem cost-effective to have two separate groups? Perhaps we could revise the charter of the AFC to cover this?

AFC's scope as presently chartered includes providing assessments of legal and financial risk facing the Coop. While much of AFC's scope is retrospective and provides review of periodic reporting, financial controls, and exposures to liability of all kinds, there is clearly a forward-looking aspect to risk assessment that pertains to evaluating new opportunities. Risk assessment though, does not describe the full range of strategic concerns and skills the Coop should employ when considering among paths forward. 

Risks should be assessed ruthlessly and balanced against equally diligent assessments of the rewards promised by any prospect.  In my view, insightful assessments of risk and reward draw from rather different regions of cognitive preoccupation and technique. For this reason, I would recommend leaving the AFC's scope concentrated on risk as presently drafted.  The best available risk assessment should contribute to the considerations of a 'quick-response executive team' but not be the driver, nor should its practices be conflated or diluted among other valid concerns. 

Present AFC Charter:  https://github.com/rchain/audit-finance-committee

**Comment by justyhoo** [comment]
There is no doubt we need to have an active Governance Committee and AFC for the Coop to properly function. Ideally, these committees would be separate. However, given the current dysfunction, I think we should consider temporarily consolidating/amalgamating the function/s  of these committees into the Executive Committee with different emphasis on roles of each committee member in the Executive Committee, including roles that cover the governance and AFC functions. This would ensure there is a temporary decision making entity acting in the members’ best interests that is properly respected by the Board/executive.  It seems to me, while the Coop is in poor health, that this type of temporary structure is more likely to authoritatively engage with the limited Board members/executive, as distinct from 3 different committees (Executive, Governance and AFC) attempting to liaise with an already stretched Board/executive. Thoughts?

**Comment by nmeyne** [comment]
Makes perfect sense Justin.  When energy and goodwill is scarce it makes sense to consolidate and focus.  Issues to address would be:

1. Maintaining adequate separation of duties and executive / supervisory separation on key decisions...  especially finances and deals

2. Empowering this consolidated exec committee to balance against President and Board without creating deadlock.

I think that's possible, especially if non-executive directors, with reputation and independence are invited to serve on the main board alongside the exec members and president.  That would be a more 'normal' board, where most business gets decided by an executive committee, not the board, who have oversight, not day to day control.

Also 'Member voice' could be heard through member observer roles and transparency on proposals (see GP001).

**Comment by justyhoo** [comment]
Thanks Nick, I agree that we need to work through those issues and that such a structure with those safeguards is achievable.

**Comment by dckc** [comment]
I lean toward having all the committees chartered but economizing by, for example, having joint meetings.

**Comment by nmeyne** [comment]
Yes it's about distinct roles and accountabilities rather than separate committees.  All possible, but the transfer of power from the president is needed to avoid it becoming a talking shop. The presidential role needs a rewrite.

**Comment by dckc** [comment]
@nmeyne , John B. (handle?),

Have you given any thought to specific wording for a proposal here? Something that would go in the bylaws and executive committee charter? Would you, please?

**Comment by nmeyne** [comment]
Working on this now.  Please see the spreadsheet https://docs.google.com/spreadsheets/d/1QkfB8rhTeQZ-KgWnqjl-B5VACtyMwQYtXuhId4TC0Uk/edit#gid=0
for supporting material (incl illustration) to support the proposal.   The draft / summary below is based on @justyhoo 's excellent suggestion above.

**Proposal:**  

To (temporarily) consolidate/amalgamate the functions of the AFC and Governance Committees into a re-constituted Executive Committee and to restrict the executive powers of the President and Board of Directors.

Functions of AFC and Governance will be performed by specific member roles and accountabilities in the new Executive Committee, including (but not limited to) additional key roles such as Engineering/Technical Strategy, Community Observer(s) and People / HR.

The role of the new Executive Committee is to implement the Board's fiduciary, strategic, and generative plans, policies, and decisions consistent with the Cooperative's vision, mission, and guiding principles.

The Chief Operating Officer chairs the Executive Committee and reports to the Board of Directors, attending Board Meetings, with other Executive Committee members, according to the agenda.

The President may also attend and serve at Executive Committee, representing the Board of Directors, and has a vote, but no overriding authority.

The Board of Directors, normally chaired by the President, will cease to engage in day to day executive decisions and will guide the Executive Committee solely by:
- defining the purpose/mission of the coop
- defining values by which the coop will perform its daily duties
- identifying the stakeholders relevant to the coop
- commissioning a strategy (from the Exec Committee) and approving it
- ensuring implementation of this strategy and supervision of the performance of the Executive Committee, using independent advice, stakeholder consultation and/or audit as necessary.

Does that work?

**Comment by justyhoo** [comment]
Bravo Nick, my view is that your proposal works. Most importantly, it would bestow custodianship of the coop back to the membership.

**Comment by nmeyne** [comment]
Adding a comment/build here on behalf of John Bodin:

1. Co-sign-off by the CEO and COO is required on any contract significantly impacting the Coop's financial or reputational standing and only after that contract has been approved by a 2/3rds majority of the Executive Committee. 

2. The institution of an Ombudsman role to be paid but non-voting that would have Observer status at all committees and the Board.

**Comment by nmeyne** [comment]
To John's comment  1. I would add that GP001 provides details of the structure and content of Board and Executive Committee proposals, criteria for significance and a process for handling proposals.

On comment 2. I would add that perhaps one of the member observer roles at each committee could fulfil the role of the independent Ombudsman and could operate the appeals proposal process as also defined in GP001.

**Comment by nmeyne** [comment]
And also (perhaps this should not need to be said, because it is so fundamental and obvious): The executive committee and board will not approve contracts, funding partnerships or commercial relationships that constrain the appointments or tenure of staff or conflict with the principles and mission of the Coop.

Note:  That applies to both the retention (as per the Thrust Capital deal) and the removal of staff (as per other rumoured deals).

**Comment by justyhoo** [comment]
As to John Bodin’s comment, I would add that such co-sign-off by CEO and COO must not be unreasonably withheld.

**Comment by nmeyne** [comment]
Further consolidation of above comments on the Proposal:
"
To (temporarily) consolidate/amalgamate the functions of the AFC and Governance Committees into a re-constituted Executive Committee and to restrict the executive powers of the President and Board of Directors.

Functions of AFC and Governance will be performed by specific member roles and accountabilities in the new Executive Committee, including (but not limited to) additional key roles such as Engineering/Technical Strategy, Community Observer(s) and People / HR.

The role of the new Executive Committee is to implement the Board's fiduciary, strategic, and generative plans, policies, and decisions consistent with the Cooperative's vision, mission, and guiding principles.

The Chief Operating Officer chairs the Executive Committee and reports to the Board of Directors, attending Board Meetings, with other Executive Committee members, according to the agenda.

The President may also attend and serve at Executive Committee, representing the Board of Directors, and has a vote, but has no other overriding authority.

The Board of Directors, normally chaired by the President, will cease to engage in day to day executive decisions and will guide the Executive Committee solely by:
defining the purpose/mission of the coop
defining values by which the coop will perform its daily duties
identifying the stakeholders relevant to the coop
commissioning a strategy (from the Exec Committee) and approving it
ensuring implementation of this strategy and supervision of the performance of the Executive Committee, using independent advice, stakeholder consultation and/or audit as necessary.

Co-sign-off by the President and COO is required on any contract significantly impacting the Coop's financial or reputational standing and only after that contract has been approved by a 2/3rds majority of the Executive Committee.  Co-sign-off by President and COO must not be unreasonably withheld.  GP001 provides details of the required structure and content of Board and Executive Committee proposals, criteria for significance and a process for handling proposals.

The executive committee and board will not approve contracts, funding partnerships or commercial relationships that constrain the appointments or tenure of staff or conflict with the principles and mission of the Coop.

A new Ombudsman role (non-voting) would have Observer status at all committees and the Board.  Member observer roles at each committee could fulfil the role of the Ombudsman and could operate the appeals proposal process as also defined in GP001.
"
Please review.

**Comment by dckc** [comment]
Why mention the AFC and governance committees at all? It's not clear to me what that means. I suggest leaving it out for simplicity / clarity ("4 corners").

Is the Ombudsman section essential? It seems like a straightforward separate issue.

"specific member roles"? which roles? This doesn't pass the "4 corners" test, to me.

strike "new" from "The role of the new Executive Committee is ..."; that's already its stated role.

"day to day executive decisions"? what does that mean, exactly? Which decisions are "day to day" and which are not? I suggest striking this.

I would strike "ensuring implementation of this strategy " too; that sounds too much like carrying it out, which I think you mean to be the role of the executive committee.

re "approved by a 2/3rds majority" -- what are the time constraints on that? How quickly can the executive committee approve something? Can it do so by email? Or only at meetings? If only at meetings, how often does it meet? When is the first time the committee could make such a decision? Committee members are nominated by the nominating committee and approved by the board, to date; do you implicitly expect to keep that process?

**Comment by dckc** [comment]
I'm not aware of offers to participate as ombudsman.

**Comment by nmeyne** [comment]
> Why mention the AFC and governance committees at all? It's not clear to me what that means. I suggest leaving it out for simplicity / clarity ("4 corners").

Audit/Finance and Governance should be distinct roles in this committee and in due course, to pass this 4 corners thing (self-standing document?) we'd need to copy in most of the content of those two charters.

> Is the Ombudsman section essential? It seems like a straightforward separate issue.

John suggested it.  We've discussed it at Governance Committee before, so it seemed appropriate to include it with the established community observer role.

> "specific member roles"? which roles? This doesn't pass the "4 corners" test, to me.

The 'member for Audit and Finance' and the 'member for Governance'.

> strike "new" from "The role of the new Executive Committee is ..."; that's already its stated role.

would 'revised' do?  The charter seemed still incomplete at June 2018?

> "day to day executive decisions"? what does that mean, exactly? Which decisions are "day to day" and which are not? I suggest striking this.

Ok but I suggest the intent is clear. Please refer to the differences between the scope of the Board of Directors and that of the Executive Committee

> I would strike "ensuring implementation of this strategy " too; that sounds too much like carrying it out, which I think you mean to be the role of the executive committee.

'assuring' ?

> `re` "approved by a 2/3rds majority" -- what are the time constraints on that? How quickly can the executive committee approve something? Can it do so by email? Or only at meetings? If only at meetings, how often does it meet? When is the first time the committee could make such a decision? Committee members are nominated by the nominating committee and approved by the board, to date; do you implicitly expect to keep that process?

Detail for the charter if this gets anywhere?

**Comment by dckc** [comment]
> Audit/Finance and Governance should be distinct roles in this committee and in due course, to pass this 4 corners thing (self-standing document?) we'd need to copy in most of the content of those two charters.

Really? All that is essential to this proposal? What I hear mostly is that this is the only essential point:

 - Co-sign-off by the President and COO is required on any contract significantly impacting the Coop's financial or reputational standing and only after that contract has been approved by a 2/3rds majority of the Executive Committee. 

Perhaps some details about how the membership of the executive committee is defined are essential. But anything beyond that is a distraction from the near-term proposal; it can be worked out later.

> Detail for the charter if this gets anywhere?

It's necessary and sufficient that this gets passed by the board, no?

**Comment by dckc** [comment]
The reason I ask "How quickly can the executive committee approve something?" is that there is no provision for doing business other than by 2/3rds majority of the committee. In #11, I didn't go so far as to say that items not on the agenda in advance are not allowed; I just said that they're subject to appeal.

**Comment by nmeyne** [comment]
I think on its own, co-sign off won't do if a President is all powerful and can appoint a COO of choice unilaterally.  I think John was suggesting the co-sign off on deals as a specific build on the whole proposal to  balance presidential/board powers and the executive team.

**Comment by dckc** [comment]
The bylaws already say that the board appoints officers, not the President.

This all powerful President myth is just that. A myth.

**Comment by dckc** [comment]
Yes, the president can appoint additional officers, and I guess he created the office of  COO.

That's perhaps worth changing in due course, but it looks manageable in the short term, to me.

In any case, I do think a proposal that is complete with just two or three paragraphs at most is most likely to be effective.

**Comment by nmeyne** [comment]
>How quickly can the executive committee approve something?

I think if a proposal is clear and understood (written) it could be signed off by correspondence or in a chat?  Within a day?  Not sure.

**Comment by dckc** [comment]
Within a day sounds fast enough. It implies a high time commitment from
executive committee members, but I guess we knew that.

To date, committee work has been all volunteer. Perhaps in due course,
executive committee members should be exempt from that and allowed to
negotiate compensation.

**Comment by nmeyne** [comment]
>proposal that is complete with just two or three paragraphs at most is most likely to be effective

OK Dan. I will look again sometime later at the condensing and wordsmithing.  I was originally  responding to your request to write up a proposal in response to the issues, adding in builds and suggestions from @justyhoo and John Bodin.

**Comment by nmeyne** [comment]
Co-sign-off by the President and COO is required on any contract significantly impacting the Coop's financial or reputational standing and only after that contract has been approved by a 2/3rds majority of the Executive Committee.
- Co-sign-off by President and COO must not be unreasonably withheld.
- Governance Process [GP001](https://docs.google.com/document/d/1nBRLjw8tuQd05GIacIhrN2FwFPJInwaiLNwiW0nb-E4/edit?usp=sharing) (the implementation of IoB#4, required for this proposal) provides details of the required structure and content of Board and Executive Committee proposals, criteria for significance and a process for handling proposals.
- The remit and composition for the Executive Committee and COO and their relationship to the Board of Directors and President are defined in the '[SM19a Board-Exec Composition and Relationships](https://docs.google.com/spreadsheets/d/1QkfB8rhTeQZ-KgWnqjl-B5VACtyMwQYtXuhId4TC0Uk/edit?usp=sharing)' document.  (required for this proposal).  PDF extract here
[RChain Coop Governance Landscape - SM19a Board-Exec Composition and Relationships.pdf](https://github.com/rchain-community/sm19a/files/3173333/RChain.Coop.Governance.Landscape.-.SM19a.Board-Exec.Composition.and.Relationships.pdf)

**Comment by dckc** [comment]
The board approved a transition plan in its [November 6, 2018 meeting](https://github.com/rchain/board/tree/master/2018/11-06):

> WHEREAS, the Cooperative has received input from the community expressing a desire to see Greg Meredith step down from his role as president,
> RESOLVED, after mainnet is launched and after Mercury backlog is completed and before Venus feature development begins in earnest, Greg Meredith will step down as President and transition to another relationship with the Cooperative in support of the development of Venus to be negotiated. Kenny Rowe will be interim President when Greg Meredith steps down. The board will initiate a process of phasing out the office of president in favor of a council of VPs: Engineering, Marketing, Token Economics/Finance, Governance. (git rev 0675dd4)

The definition of the "Mercury backlog" remains unclear.

ref subsequent discussion in the [2019-03-05 board meeting](https://github.com/rchain/board/tree/master/2019/03-05)

[current board repo git rev is a0617db Oct 24, 2020](https://github.com/rchain/board/commit/a0617db4308a7d1465519180bfff3b10e9fb9b81)
